jdaviz/configs/default/plugins/subset_plugin/utils.py

```diff
@@ -0,0 +1,175 @@
+# utility functions for subset plugin
+# these don't belong in glue.core.region_translators, as they deal with conversions that
+# only happen within this plugin (e.g between region/'subset defintion')
+
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from jdaviz.core.region_translators import regions2roi
+import numpy as np
+from regions import CircleAnnulusSkyRegion, CircleSkyRegion, EllipseSkyRegion, RectangleSkyRegion
```

Comment:
Order imports to be compatible with isort.

```suggestion
import numpy as np
from astropy import units as u
from astropy.coordinates import SkyCoord
from regions import CircleAnnulusSkyRegion, CircleSkyRegion, EllipseSkyRegion, RectangleSkyRegion

from jdaviz.core.region_translators import regions2roi
```